src/bokeh/core/property/descriptors.py

```diff
@@ -144,11 +147,15 @@ class AliasPropertyDescriptor(Generic[T]):
 
     serialized: bool = False
 
-    def __init__(self, name: str, aliased_name: str, property: Property[T]) -> None:
+    @property
+    def aliased_name(self) -> str:
+        return self.alias.aliased_name
+
+    def __init__(self, name: str, alias: Alias[T]) -> None:
         self.name = name
-        self.aliased_name = aliased_name
-        self.property = property
-        self.__doc__ = f"This is a compatibility alias for the ``{aliased_name}`` property"
+        self.alias = alias
+        self.property = alias
```

Comment:
What is the rationale for assigning the same value to two different attributes?

Reply:
`property` is needed to make `Alias` to fulfill bokeh's descriptor interface. The problem with having a `property` attribute is that you can't do anything with on the class definition level, i.e. this:
```py
class Some:
    property: str
    @property
    def some_getter() -> int:
        return 0
```
will fail, because now local `property` overshadows built-in `property`. This is ofcourse solvable with some name mangling, but for the purpose of this PR it was easier to just introduce a new name `alias` locally. I need to be able to reference the property on the class definition level, because I need to override the type of the property in derived classes.

It's not the first time I had to deal with this issue (you can see commented out `property: Property[T]` in `PropertyDescriptor`) and I will be fixing this properly at some point, probably by just renaming property to something else.